client/web/src/components/fuzzyFinder/HighlightedLink.tsx

```diff
@@ -36,33 +36,37 @@ export function offsetSum(props: HighlightedLinkProps): number {
 export const HighlightedLink: React.FunctionComponent<HighlightedLinkProps> = props => {
     const spans: JSX.Element[] = []
     let start = 0
-    function pushSpan(className: string, startOffset: number, endOffset: number): void {
+    function pushElement(kind: 'mark' | 'span', startOffset: number, endOffset: number): void {
         if (startOffset >= endOffset) {
             return
         }
         const text = props.text.slice(startOffset, endOffset)
         const key = `${startOffset}-${endOffset}`
-        const span = (
-            <span key={key} className={className}>
-                {text}
-            </span>
-        )
-        spans.push(span)
+        if (kind === 'mark') {
+            spans.push(
+                <mark key={key} className={styles.mark}>
```

Comment:
Is this class name needed? I think `<mark>` should have global styles
```suggestion
                <mark key={key}>
```

Reply:
There doesn't appear to be a global style for `<mark>`, I removed the class and it looks like this now. It's not terrible in light theme, but difficult to read in dark theme

![CleanShot 2021-06-09 at 16 25 01](https://user-images.githubusercontent.com/1408093/121374243-08f07b80-c940-11eb-81ab-7e2862c14fea.png)
![CleanShot 2021-06-09 at 16 25 05](https://user-images.githubusercontent.com/1408093/121374239-0857e500-c940-11eb-8374-d1bceb4dab08.png)

Reply:
I kept the custom class since it looks better

Reply:
Interesting, that's probably something we should fix @sourcegraph/frontend-platform (not blocking for this PR)

Reply:
Interestingly, in search results we don't use `<mark>` either, but `<span class="selection-highlight">`.

Reply:
Yeah we should fix that some day ideally 😄

Reply:
At least the variable is already named `--mark-bg`